argilla/docs/how_to_guides/migrate_from_legacy_datasets.md

```diff
@@ -1 +1,287 @@
+            responses.append(rg.Response(question_name="label", value=annotation, user_id=user_id))
+    
+        vectors = (data.get("vectors") or {})
+        return rg.Record(
+            id=data["id"],
+            fields=data["inputs"],
+            # The inputs field should be a dictionary with the same keys as the `fields` in the settings
+            metadata=data["metadata"],
+            # The metadata field should be a dictionary with the same keys as the `metadata` in the settings
+            vectors=[rg.Vector(name=name, values=value) for name, value in vectors.items()],
+            suggestions=suggestions,
+            responses=responses,
+        )
+    ```
+
+=== "For multi-label classification"
+
+    ```python
+    def map_to_record_for_multi_label(data: dict, users_by_name: dict, current_user: rg.User) -> rg.Record:
+        """ This function maps a text classification record dictionary to the new Argilla record."""
+        suggestions = []
+        responses = []
+        
+        if prediction := data.get("prediction"):
+            labels, scores = zip(*[(pred["label"], pred["score"]) for pred in prediction])
+            agent = data["prediction_agent"]
+            suggestions.append(rg.Suggestion(question_name="labels", value=labels, score=scores, agent=agent))
+    
+        if annotation := data.get("annotation"):
+            user_id = users_by_name.get(data["annotation_agent"], current_user).id
+            responses.append(rg.Response(question_name="label", value=annotation, user_id=user_id))
+    
+        vectors = data.get("vectors") or {}
+        return rg.Record(
+            id=data["id"],
+            fields=data["inputs"],
+            # The inputs field should be a dictionary with the same keys as the `fields` in the settings
+            metadata=data["metadata"],
+            # The metadata field should be a dictionary with the same keys as the `metadata` in the settings
+            vectors=[rg.Vector(name=name, values=value) for name, value in vectors.items()],
+            suggestions=suggestions,
+            responses=responses,
+        )
+    ```
+    
+=== "For token classification"
+
+    ```python
+    def map_to_record_for_span(data: dict, users_by_name: dict, current_user: rg.User) -> rg.Record:
+        """ This function maps a token classification record dictionary to the new Argilla record."""
+        suggestions = []
+        responses = []
+    
+        if prediction := data.get("prediction"):
+            scores = [span["score"] for span in prediction]
+            agent = data["prediction_agent"]
+            suggestions.append(rg.Suggestion(question_name="labels", value=prediction, score=scores, agent=agent))
+    
+        if annotation := data.get("annotation"):
+            user_id = users_by_name.get(data["annotation_agent"], current_user).id
+            responses.append(rg.Response(question_name="spans", value=annotation, user_id=user_id))
```

Comment:
`question_name` is different for `rg.Suggestion` and `rg.Response`, it should be the same, whether that's `labels` or `spans`.